crates/release_plz_core/src/command/release_pr/git.rs

```diff
@@ -0,0 +1,348 @@
+use anyhow::{Context, Result, bail};
```

Comment:
```suggestion
use anyhow::{Context, bail};
```
please don't import `anyhow::Result`, I don't like it.